pkg/controller/networkpolicy/validate.go

```diff
@@ -513,12 +513,24 @@ func (v *antreaPolicyValidator) validatePeers(ingress, egress []crdv1alpha1.Rule
 		return "", true
 	}
 	for _, rule := range ingress {
+		if rule.To != nil || rule.ToServices != nil {
```

Comment:
the openAPI schema takes care of ths case right?

Reply:
You right. Removed.